apps/trackers/jira/query.py

```diff
@@ -128,6 +128,14 @@ def generate_labels(self):
             )
         )
 
+        # If all affects are NEW and if this tracker is new, add label validation-requested.
+        if not self.tracker.external_system_id:
+            affect_states = list(
+                self.tracker.affects.all().values_list("affectedness", flat=True)
+            )
+            if set(affect_states) == set([Affect.AffectAffectedness.NEW]):
+                self._query["fields"]["labels"] += ["validation-requested"]
+
```

Comment:
I think this could be more efficient and less verbose if we leverage querysets:

```suggestion
        # If all affects are NEW and if this tracker is new, add label validation-requested.
        if self.tracker.external_system_id or self.tracker.affects.exclude(affectedness=Affect.Affectedness.NEW).exists():
            return
        self._query["fields"]["labels"] += ["validation-requested"]
```